test/integ/quote-classic.test.ts

```diff
@@ -584,8 +584,9 @@ describe('quote', function () {
         });
         const native = NATIVE_CURRENCY[chain];
         it(`${native} -> erc20`, async () => {
-          if (chain === ChainId.BLAST) {
+          if (chain === ChainId.BLAST || chain === ChainId.ZORA) {
```

Comment:
It seems like this test is swapping from ETH -> WETH (`TEST_ERC20_2`) so would still work for Zora

Reply:
Actually there's a nuance for wrap/unwrap. When FE shows wrap/unwrap, FE submits through universal router wrap/unwrap command directly (https://github.com/Uniswap/universal-router/blob/main/contracts/libraries/Commands.sol#L31). This means FE will not invoke the routing backend for wrap/unwrap.

Also, because at V2 and V3 protocol layers, pools don't support native currency, so offline routing will automatically wrap the native currency if it detects one, so routing will end up quoting WETH -> WETH, and thus getting the error [INVARIANT ADDRESS](https://github.com/Uniswap/sdks/blob/main/sdks/sdk-core/src/entities/token.ts#L79) from sdk-core.

Reply:
Ah I see. Can you mention that in a comment?